modules/nuxt-pdf.yml

```diff
@@ -0,0 +1,15 @@
+name: nuxt-pdf
+repo: sidebase/nuxt-pdf
+npm: '@sidebase/nuxt-pdf'
+icon: ''
+github: https://github.com/sidebase/nuxt-pdf
+website: https://sidebase.io/nuxt-pdf
+learn_more: ''
+category: Devtools
```

Comment:
```suggestion
category: Extensions
```

Reply:
Done (: But CI is dfailing for reasons I do not understand - "cannot edit PR"?

Maybe because it's from a fork?